deploy-helm/helm-operator-deployment.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: flux-helm-operator
+  # helm-operator should be created in the same namespace
+  # as flux agent
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        name: flux-helm-operator
+    spec:
+      serviceAccount: flux
+      volumes:
+      - name: git-key
+        secret:
+          secretName: flux-git-deploy
+      containers:
+      - name: flux-helm-operator
+        # There are no ":latest" images for helm-operator. Find the most recent
+        # release or image version at https://quay.io/weaveworks/flux (both have the same latest tag)
```

Comment:
This needs altering so it doesn't refer to the flux daemon image repo.

Reply:
a good spot. Will correct